src/main/resources/static_config/static_roles.yml

```diff
@@ -83,6 +83,10 @@ kibana_server:
   - "cluster_monitor"
   - "cluster_composite_ops"
   - "cluster:admin/xpack/monitoring*"
+  - "cluster:admin/opendistro/reports/instance/poll"
```

Comment:
Can you describe each of these permissions?

Reply:
- "cluster:admin/opendistro/reports/instance/poll" --> Get scheduled report instance information (metadata from configuration index).
  - "cluster:admin/opendistro/reports/instance/get" --> get information of specific report instance
  - "cluster:admin/opendistro/reports/instance/update_status" --> change status of report instance from scheduled to success/failure
  - "cluster:admin/opendistro/notifications/send" --> send notification mail to email ids configured in report instance